tools/sk-libfido2.h

```diff
@@ -0,0 +1,55 @@
+#ifndef THIRD_PARTY_LIBFIDO2_TOOLS_SK_LIBFIDO2_H_
+#define THIRD_PARTY_LIBFIDO2_TOOLS_SK_LIBFIDO2_H_
+
+#include <stdlib.h>
+#include <stdint.h>
+
+#define SK_VERSION_MAJOR	0x00020000 /* current API version */
+
+/* Return values */
+#define SK_SUCCESS 0
+#define SK_FAIL    1
+
+/* Flags */
+#define TUP_FLAG             0x01
+#define INDIVIDUAL_CERT_FLAG 0x80
+#define SK_USER_PRESENCE_REQD	0x01
```

Comment:
TUP_FLAG and INDIVIDUAL_CERT_FLAG are currently unused, so I can only infer their purpose from context. The difference between TUP_FLAG and SK_USER_PRESENCE_REQD is particularly unclear to me. I suggest adding these new flags along with the code using them.